src/librustc_mir/hair/pattern/check_match.rs

```diff
@@ -204,25 +204,42 @@ impl<'a, 'tcx> MatchVisitor<'a, 'tcx> {
             // is uninhabited.
             let pat_ty = self.tables.node_id_to_type(scrut.hir_id);
             let module = self.tcx.hir().get_module_parent(scrut.id);
+            let mut def_span = None;
+            let mut missing_variants = vec![];
```

Comment:
I usually write empty vectors as `Vec::new()` (thinking, "no need to the macro sugar when there aren't any elements to initialize"), but `rustfmt` doesn't seem to have an opinion. (I guess you could argue that it's technically a semantic change.)